fapi-client/src/main/scala/com/gu/facia/api/utils/FaciaContentUtils.scala

```diff
@@ -144,7 +144,9 @@ object FaciaContentUtils {
   )
   def maybeShortUrl(fc: FaciaContent) = fieldsGet(fc)(_.flatMap(_.shortUrl))
   def shortUrl(fc: FaciaContent): String = maybeShortUrl(fc).getOrElse("")
-  def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(_.replaceFirst("^[a-zA-Z]+://gu.com", ""))
+  def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(
+    _.replaceFirst("^https?://gu.com", "")
+      .replaceFirst("^https?://(www.)+theguardian.com", ""))
```

Comment:
Might be unnecessary to make the `www.` optional as well? Plus do we need to escape any of the other characters i.e. `/` and `.`?

Reply:
actually, sorry, you're not making it optional with `+`?

Reply:
Ah yeah this should have been `?` to make it optional. I guess it does make more sense to actually have it as a required part of the string given that this is parsing what is sent over from CAPI directly.